.golangci.yml

```diff
@@ -11,6 +11,8 @@ linters:
     - whitespace
     - gocognit
     - gomnd
+    - godot
+    - goerr113
```

Comment:
Might be good to create a tracking issue for going back and cleaning up `goerr113` related issues.